llvm/lib/Target/AMDGPU/SIRegisterInfo.h

```diff
@@ -64,6 +64,8 @@ class SIRegisterInfo final : public AMDGPUGenRegisterInfo {
     bool hasReg() { return VGPR != 0; }
   };
 
+  const GCNSubtarget &getSubtarget() const { return ST; }
```

Comment:
Shouldn't need this, the pass already has the subtarget

Reply:
removed